packages/rx_bloc_cli/lib/src/processors/android/app_build_gradle_processor.dart

```diff
@@ -0,0 +1,192 @@
+      final end = start + content.length;
+      buffer.replaceRange(start, end, replacement);
+    }
+
+    void _replaceQuotedVal(
+      String content,
+      String replacement, {
+      int start = 0,
+    }) {
+      final sIndex = buffer.nthIndexOf('"',
+              start: buffer.nthIndexOf(content, start: start)) +
+          1;
+      final eIndex = buffer.nthIndexOf('"', start: sIndex);
+      buffer.replaceRange(sIndex, eIndex, replacement);
+    }
+
+    _replaceVal('flutter.compileSdkVersion', '$compileSDK');
+    _replaceVal('flutter.targetSdkVersion', '$targetSDK');
+    _replaceVal('flutter.minSdkVersion', '$minSDK');
+
+    _replaceQuotedVal('namespace', _packageId);
+    _replaceQuotedVal(
+      'applicationId',
+      _packageId,
+      start: buffer.nthIndexOf('defaultConfig'),
+    );
+  }
+
+  void _adjustBuildTypesAndSigningConfigs(StringBuffer buffer) {
+    final sIndex = buffer.nthIndexOf('buildTypes') - 4;
+    final eIndex = buffer.nthIndexOf('}', n: 2, start: sIndex) + 1;
+
+    const content = '''
+    signingConfigs {
+        if (System.getenv("ANDROID_KEYSTORE_PATH")) {
+            release {
+                storeFile file(System.getenv("ANDROID_KEYSTORE_PATH"))
+                keyAlias System.getenv("ANDROID_KEYSTORE_ALIAS")
+                keyPassword System.getenv("ANDROID_KEYSTORE_PRIVATE_KEY_PASSWORD")
+                storePassword System.getenv("ANDROID_KEYSTORE_PASSWORD")
+            }
+        } else {
+            release {
+                keyAlias keystoreProperties['keyAlias']
+                keyPassword keystoreProperties['keyPassword']
+                storeFile keystoreProperties['storeFile'] ? file(keystoreProperties['storeFile']) : null
+                storePassword keystoreProperties['storePassword']
+            }
+        }
+    }
+    
+    buildTypes {
+        release {
+            signingConfig signingConfigs.release
+            minifyEnabled true
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+        debug {
+            signingConfig signingConfigs.debug
+        }
+    }
+    ''';
+
+    buffer.replaceRange(sIndex, eIndex, content);
+  }
+
+  void _addKeyPropertiesConfig(StringBuffer buffer) {
+    const beforePattern = 'def localProperties = new Properties()';
+    const content = '''
+def keystoreProperties = new Properties()
+def keystorePropertiesFile = rootProject.file('key.properties')
+if (keystorePropertiesFile.exists()) {
+    keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
+}
+
+''';
+    buffer.insertBefore(beforePattern, content);
+  }
+
+  void _buildDependenciesList(StringBuffer buffer) {
+    final sIndex = buffer.nthIndexOf('dependencies');
+    final eIndex = buffer.nthIndexOf('}', start: sIndex) + 1;
+
+    var content = 'dependencies {\n';
+    if (args.patrolTestsEnabled) {
+      content +=
+          '${_tabSpace}androidTestUtil "androidx.test:orchestrator:1.4.2"\n';
```

Comment:
Do we really need to hardcode the version of the orchestrator ?

Reply:
Yes, as it is required by the patrol package and is documented in the [official patrol setup](https://patrol.leancode.pl/getting-started#integrate-with-native-side). 

Also, omitting a version results in build errors.